src/microbe_stage/MicrobeHUD.cs

```diff
@@ -838,4 +838,25 @@ private void OnTranslationsChanged()
         UpdateColonySizeForMulticellular();
         UpdateColonySizeForMacroscopic();
     }
+
+    private void ToggleProcessPressed(ChemicalEquation equation)
+    {
+        if (!stage!.HasAlivePlayer || !stage!.Player.Has<BioProcesses>())
+            return;
+
+        ref var processes = ref stage.Player.Get<BioProcesses>();
+
+        if (processes.ActiveProcesses == null)
+            return;
+
+        for (int i = 0; i < processes.ActiveProcesses.Count; ++i)
+        {
+            if (processes.ActiveProcesses[i].Process.Name == equation.EquationFromProcess!.Name)
```

Comment:
For safety this method should check if equation has no process set and error out with a GD.PrintErr around line 846. Also comparing processes by name is a really bad way to compare them so something else should be done. Perhaps the EquationFromProcess needs to be extended to have a method to check if the underlying process matches.